docs/source/install.rst

```diff
@@ -31,13 +31,7 @@ To get started with MyoSuite, clone this repo with pre-populated submodule depen
 
    git clone --recursive https://github.com/facebookresearch/myosuite.git
    cd myosuite
-   pip install -e -r requirements
-
-OR Add repo to pythonpath by updating `~/.bashrc` or `~/.bash_profile`
-
-.. code-block:: bash
-
-   export PYTHONPATH="<path/to/myosuite>:$PYTHONPATH"
+   pip install -e -r .
```

Comment:
double-check this. I believe -r option is when you are specifying a file for requirements.

Reply:
fixed in the last commit